backend/src/sample_entries.rs

```diff
@@ -0,0 +1,89 @@
+use num_bigint::ToBigUint;
+use summa_solvency::merkle_sum_tree::Entry;
+
+pub fn get_sample_entries() -> Vec<Entry<2>> {
```

Comment:
remove this file

Reply:
Removed at [5a9120f](https://github.com/summa-dev/summa-solvency/pull/179/commits/5a9120f25dfebf26826ba5799e4a53c571e8cf06)